tests/unit/moduleapi/test_lazyfree.tcl

```diff
@@ -0,0 +1,27 @@
+set testmodule [file normalize tests/modules/test_lazyfree.so]
+
+start_server {tags {"modules"}} {
+    r module load $testmodule
+
+    test "modules allocated memory can be reclaimed in the background" {
+        set orig_mem [s used_memory]
+
+        # LAZYFREE_THRESHOLD is 64
+        for {set i 0} {$i < 65} {incr i} {
+            r lazyfreelink.insert lazykey $i
+        }
+
+        assert {[r lazyfreelink.len lazykey] == 65}
+
+        set peak_mem [s used_memory]
+        assert {[r unlink lazykey] == 1}
+        assert {$peak_mem > $orig_mem+65}
```

Comment:
I think this test might be flaky working at a byte level. I would consider allocating more substantial amount of memory in the test module (e.g. 500KB per entry or so) so it the effect would be more observable and robust.

Reply:
Yes, I directly refer to'lazyfree.tcl' and change the memory allocation amount to 100000, which has far exceeded LAZYFREE_THRESHOLD. At the same time, because of the'lazyfreed_objects' information, I can be sure that the memory of this module is lazyfreed